backend-api/tests/api-tests/clientCredentialsGrantFlow.test.ts

```diff
@@ -42,14 +58,110 @@ describe("POST /token", () => {
       expect(response.status).toBe(400);
     });
   });
+
+  describe("Given the grant_type value is invalid", () => {
+    it("Returns a 400 Bad Request response", async () => {
+      const response = await axiosInstance.post(
+        `${apiBaseUrl}/async/token`,
+        "grant_type=invalid_grant_type",
+        {
+          headers: {
+            "x-custom-auth": "Basic " + toBase64(clientIdAndSecret),
+          },
+        },
+      );
+
+      expect(response.data).toStrictEqual({
+        error: "invalid_grant",
+        error_description: "Invalid grant type or grant type not specified",
```

Comment:
API spec says this `error_description`  should be `Invalid grant` - how do we want to proceed?